documentation/porting_examples/example5.md

```diff
@@ -0,0 +1,121 @@
+# Transfer example
```

Comment:
Could you provide a general overview of what the script does here? Also, it may be worth renaming the file to something more descriptive